test/cri-containerd/runpodsandbox_test.go

```diff
@@ -721,3 +723,151 @@ func Test_RunPodSandbox_PortMappings_LCOW(t *testing.T) {
 	}
 	runPodSandboxTest(t, request)
 }
+
+func Test_RunPodSandbox_CustomizableScratchDefaultSize_LCOW(t *testing.T) {
+	pullRequiredLcowImages(t, []string{imageLcowK8sPause})
+
+	annotations := map[string]string{
+		"io.microsoft.virtualmachine.computetopology.memory.allowovercommit": "true",
+	}
+
+	output, errorMsg, exitCode := createSandboxContainerAndExec(t, annotations)
+
+	if exitCode != 0 {
+		t.Fatalf("Exec into container failed with: %v and exit code: %d, Test_RunPodSandbox_CustomizableScratchDefaultSize_LCOW", errorMsg, exitCode)
+	}
+
+	// Format of output for df is below
+	// Filesystem           1K-blocks      Used Available Use% Mounted on
+	// overlay               20642524        36  19577528   0% /
+	// tmpfs                    65536         0     65536   0% /dev
+	scanner := bufio.NewScanner(strings.NewReader(output))
+	found := false
+	var cols []string
+	for scanner.Scan() {
+		outputLine := scanner.Text()
+		if cols = strings.Fields(outputLine); cols[0] == "overlay" && cols[5] == "/" {
+			found = true
+			t.Log(outputLine)
+			break
+		}
+	}
+
+	if !found {
+		t.Fatalf("could not find the correct output line for overlay mount on / n: error: %v, exitcode: %d", errorMsg, exitCode)
+	}
+
+	// df command shows size in KB, 20642524 is 20GB
+	if cols[1] != "20642524" {
```

Comment:
Im wondering if we should test for this value within a range. +/- 10MB or so? Just dont want to fail if an extra write or something happens in the kernel

Reply:
That's a good idea. I will make that change and resend